doc/ethapp.adoc

```diff
@@ -1455,6 +1455,50 @@ None
 | Web3 Check setting status  | 1
 |====================================
 
+### SIGN EIP 7702 AUTHORIZATION
+
+#### Description
+
+This command computes the signature for an element of a EIP 7702 Authorization list (https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7702.md) on the given account, for the given delegate, chain ID and nonce.
+
+The user is prompted to confirm the operation before the signature is issued.
+
+#### Coding
+
+_Command_
+
+[width="80%"]
+|==============================================================
+| *CLA* | *INS*  | *P1*               | *P2*       | *LC*
+|   E0  |   34   | 00                 | 00         | 00
+|==============================================================
+
+_Input data_
+
+[width="80%"]
+|==========================================
```

Comment:
This should be a TLV struct, something like : 

| Name           | Tag  | Payload type |
|----------------|------|--------------|
| DERIVATION_IDX | 0x01 | uint32       |
| DELEGATE_ADDR  | 0x02 | uint8[20]    |
| CHAIN_ID       | 0x03 | uint64       |
| NONCE          | 0x04 | uint64       |

With as many DERIVATION_IDX in the struct as there are nodes in the derivation path.

So the APDU would contain (like it is described for other APDUs containing TLV structs) :

| Description   | Length   |
|---------------|----------|
| struct length | 2        |
| TLV struct    | variable |

Reply:
can do, but that'd make it quite different from similar TX / message signing APDUs, so maybe a hassle for third party integrators building from scratch. Your call.

Reply:
They could either use the new https://github.com/LedgerHQ/device-sdk-ts or draw some inspiration from it if they still want to code something from scratch.
This APDU format is closer to all the new features we've been adding recently.